File: examprep/oddeven.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
	Scanner oddeven = new Scanner(System.in);
	int num;
	System.out.println("Enter a number");
	num = oddeven.nextInt();

	if(num % 2 == 0){
	    System.out.println("Even");
	}
	else{
	    System.out.println("Odd");
	}
	oddeven.close();
    }
}

File: CSharp/oddeven.cs
using System;

namespace OddEven{
    class Odd{
	static void Main(string[] args){
	    string no;
	    Console.WriteLine("Enter a Number");

	    no = Console.ReadLine();

	    int num = Convert.ToInt32(no);

	    if(num % 2 == 0){
		Console.WriteLine("Number is even");
	    }
	    else{
		Console.WriteLine("Number is odd");
	    }
	}
    }
}
